README.md

```diff
@@ -1 +1,29 @@
-# coding-monads-starter
\ No newline at end of file
+# coding-monads-starter
+
+### to run project
+
+1. install dependencies inside root folder and client folder.
```

Comment:
Maybe we should add additional command in scripts for this to be one command to execute